M/MAGEMin/build_tarballs.jl

```diff
@@ -85,7 +85,7 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency(PackageSpec(name="NLopt_jll", uuid="079eb43e-fd8e-5478-9966-2cf3e3edb778"))
+    Dependency(PackageSpec(name="NLopt_jll", uuid="079eb43e-fd8e-5478-9966-2cf3e3edb778"); compat="2.8.0")
```

Comment:
> Fixed NLopt version to 2.8.0 as more recent version leads to large variation in calculation results between platforms

This compat follows [Pkg compat bounds syntax](https://pkgdocs.julialang.org/v1/compatibility/#Version-specifier-format).  `2.8.0` means `[2.8.0, 3)`, which I'm not sure is what you want, based on your comment.

Reply:
Ah sorry. You are right. I want to fix the specific version not put a lower bound. Thank you for providing the documentation. I fix this after lunch

Reply:
Might also be beneficial for you to work with upstream to get whatever issue you see fixed, not to get stuck with an outdated version forever.

Reply:
Yes, absolutely. This is also what I want to do. I am teaching with my code next week, and my plan was to contact NLopt author afterward.